src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java

```diff
@@ -319,7 +319,7 @@ private IAIState pickupItems()
             }
         }
 
-        incrementActionsDoneAndDecSaturation();
+        incrementActionsDone();
 
```

Comment:
at least make this a "continous action" decrement of saturation (eggs etc that are picked up are a action too)

Reply:
what are you talking about?